### PR TITLE
Added resources section + Arrow to primary front page

### PR DESCRIPTION
**Pull request by chrisdedman**
Added the section for the new website for the resources library. Additionally, I added an arrow at the bottom center of the screen for visitors to click on (or encourage scrolling). The arrow when clicked has a little transition animation down to the next section of the website.

_I also fixed some indentation and code formatting issues._
_I added comment to what has ben modify by me to avoid scrolling through the whole file (due to reformatting of the code, what I changed may be hard to read, sorry)_

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,117 +1,175 @@
...
-        </section>
-        <section id="info" class="bg-secondary">
-            <div class="container-fluid info-container d-flex flex-fill">
-                <div class="flex-fill">
-                    <div class="row">
-                        <div class="col-lg-6">
-                            <div class="d-flex justify-content-center align-items-center h-100">
-                                <img src="images/rocket.svg">
-                            </div>
+            <div class="floating-arrow">
```
Comment (chrisdedman) [inline public/index.html:67]: Added this

File: public/index.html [hunk 2]
```diff
@@ -1,117 +1,175 @@
...
-                            <div class="row flex-column align-items-center my-5">
-                                <div class="col-md-6">
-                                    <p class="title">Cursif</p>
-                                </div>
-                                <div class="col-md-6">
-                                    <p class="description"><a href="https://github.com/Code-Society-Lab/cursif">Cursif</a> is a collaborative, scriptable and flexible note taking application that aimes to help teams manage their projects.</p>
-                                </div>
+                    </div>
+                </div>
+                <!-- Resources Section -->
```
Comment (chrisdedman) [inline public/index.html:137]: Added this section

File: public/index.html [hunk 3]
```diff
@@ -1,117 +1,175 @@
...
                     </div>
                 </div>
             </div>
-        </section>
-    </body>
-</html>
+        </div>
+    </section>
+
+    <!-- JavaScript for smooth scrolling when arrow clicked -->
```
Comment (chrisdedman) [inline public/index.html:164]: Added this JS section

**Review by PenguinBoi12 — APPROVED** [review]
Looks good to me.

**Pull request merged**